recipes-tag/analysis/meta.yaml

```diff
@@ -25,9 +25,10 @@ requirements:
     - ffmpeg
     - graphviz
     - h5py >=2.8.0
-    - ipympl
+    - ipykernel==4.8.2
```

Comment:
This one doesn't affect widgets, so I think we should leave it unpinned.

Reply:
OK, thanks. Unpinned in https://github.com/NSLS-II/lightsource2-recipes/pull/466/commits/80368470b24ad17822a0072a6a8786c560e6f755.